docs/DeveloperGuide.adoc

```diff
@@ -279,104 +298,446 @@ Refer to the guide <<DevOps#, here>>.
 
 *Target user profile*:
 
-* has a need to manage a significant number of contacts
-* prefer desktop apps over other types
-* can type fast
-* prefers typing over mouse input
-* is reasonably comfortable using CLI apps
+* money-conscious NUS students
+* number of transactions are significant and needs to be managed
+* apps are preferred on desktop over other platforms
+* typists that type quickly and prefers typing over mouse input
+* CLI usage does not pose any discomfort
 
-*Value proposition*: manage contacts faster than a typical mouse/GUI driven app
+*Value proposition*:
+
+* provides a more effective financial management compared to typical mouse/GUI driven app
+* plans for future spending are made easier
+* congregates all info regarding fees and costs in a single location
 
 [appendix]
 == User Stories
 
 Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (unlikely to have) - `*`
 
-[width="59%",cols="22%,<23%,<25%,<30%",options="header",]
+[width="100%",cols="5%,<25%,<30%,<40%",options="header",]
 |=======================================================================
 |Priority |As a ... |I want to ... |So that I can...
-|`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
+|`* * *`
+|NUS student
+|track my spending
+|manage my finances better
+
+|`* * *`
+|movie addict who watches every movie that comes out
+|keep track of my movie ticket and snack expenses
+|have enough money to watch the movies I want.
+
+|`* * *`
+|computing student who has many software subscriptions
+|use the recurring expenses tracker
+|manage my subscription payments
+
+|`* * *`
+|student staying in the Halls with many expenses
+|see all my expenses in a central spot
+|keep track of all the different expenses
```

Comment:
Not specific to hall residents. Can consider making the actor generic or coming up with a hall specific user story